ckan/templates/snippets/search_form.html

```diff
@@ -45,8 +45,10 @@
   {% endblock %}
 
   {% block search_title %}
-    {% if not no_title %}
+    {% if not error %}
```

Comment:
no_title is not referred to anywhere else any more. I guess it was to indicate an error. Anyway, we should say how many results there are if SOLR didn't work in the first place.